### PR TITLE
Axel

### PR DESCRIPTION
**Pull request by aquinoax3**

### CODE REVIEW

File: __tests__/models/models.tests.js [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        nickname: "Phil",
+        email: "Phil@test.com",
+        workouts:[]
+    }
+
+    const mockWorkout = {
+        _id: workoutId,
+        type: "UpperBody",
+        level:"Intermediate",
+        focusArea: "Back",
```
Comment (Aaron-G-Sanchez) [inline __tests__/models/models.tests.js:24]: Nit: Can you get this to align with the focusAreas we will using for MVP?

File: __tests__/models/models.tests.js [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+        _id: userId,
+        userId: "1",
+        nickname: "Phil",
+        email: "Phil@test.com",
+        workouts:[]
+    }
+
+    const mockWorkout = {
+        _id: workoutId,
+        type: "UpperBody",
```
Comment (Aaron-G-Sanchez) [inline __tests__/models/models.tests.js:22]: Nit: Should just be 'upper'

File: __tests__/models/models.tests.js [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+        level:"Intermediate",
+        focusArea: "Back",
+        exercise: []
+    }
+
+    beforeAll(async () => {
+        await memoryServerConnect()
+
+        await User.create()
+        await Workout.create()
```
Comment (Aaron-G-Sanchez) [inline __tests__/models/models.tests.js:32]: I don't think you need these two lines.

File: __tests__/models/models.tests.js [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+        rep: 10,
+        set: 3
+    }
+
+    beforeAll(async () => {
+        await memoryServerConnect()
+
+        // Test Data
+        await Workout.create();
+        await Exercise.create();
```
Comment (Aaron-G-Sanchez) [inline __tests__/models/models.tests.js:120]: I don't think you need these two lines as well.

File: __tests__/models/models.tests.js [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+        _id: exerciseId,
+        name: "Push Ups",
+        rep: 10,
+        set: 8
+    }
+
+    beforeAll(async () => {
+        await memoryServerConnect()
+
+        await Exercise.create()
```
Comment (Aaron-G-Sanchez) [inline __tests__/models/models.tests.js:199]: See comments above.

**Review by Aaron-G-Sanchez — CHANGES_REQUESTED** [review]
Looking good!

File: __tests__/models/models.tests.js (hunk 5 above)
Comment (aquinoax3) [reply __tests__/models/models.tests.js:199]: Done

File: __tests__/models/models.tests.js (hunk 4 above)
Comment (aquinoax3) [reply __tests__/models/models.tests.js:120]: Done

File: __tests__/models/models.tests.js (hunk 3 above)
Comment (aquinoax3) [reply __tests__/models/models.tests.js:32]: Done

File: __tests__/models/models.tests.js (hunk 2 above)
Comment (aquinoax3) [reply __tests__/models/models.tests.js:22]: Done

File: __tests__/models/models.tests.js (hunk 1 above)
Comment (aquinoax3) [reply __tests__/models/models.tests.js:24]: Done

File: src/routes/users/UserRouter.js [hunk 6]
```diff
@@ -34,6 +34,37 @@ router.get('/', async (req, res, next) => {
...
+router.post("/users", async (req, res, next) => {
+  const user = new User(req.body);
+
+  try {
+    console.log("Received Data, ", req.body)
+    await user.save({ new: true })
+  } catch (error) {
+    res.statusCode(500).send(error) 
   }
 })
```
Comment (Aaron-G-Sanchez) [inline src/routes/users/UserRouter.js:69]: These routes won't be needed because this logic is being handled by the route above.

**Review by Aaron-G-Sanchez — CHANGES_REQUESTED** [review]

File: src/routes/users/UserRouter.js (hunk 6 above)
Comment (aquinoax3) [reply src/routes/users/UserRouter.js:69]: Done

**Review by Aaron-G-Sanchez — APPROVED** [review]

**Pull request merged**